src/pages/Bridge/IssueForm/index.tsx

```diff
@@ -378,6 +374,16 @@ const IssueForm = (): JSX.Element | null => {
             })}
           </FormTitle>
           <div>
+            <AvailableBalanceUI
+              label={t('issue_page.maximum_in_single_request')}
+              balance={displayMonetaryAmount(requestLimits.singleVaultMaxIssuable)}
+              tokenSymbol={WRAPPED_TOKEN_SYMBOL}
+            />
+            <AvailableBalanceUI
+              label={t('issue_page.maximum_total_request')}
+              balance={displayMonetaryAmount(requestLimits.totalMaxIssuable)}
+              tokenSymbol={WRAPPED_TOKEN_SYMBOL}
+            />
```

Comment:
I found a UI/UX issue that error messages overlapped the following UI element, so I updated like so (moved from below to above the main input). I also ask Jay about this UX change in the discord channel. Once they approve it, we can go with it.
<img width="1496" alt="Screenshot 2022-12-11 at 12 10 33 AM" src="https://user-images.githubusercontent.com/84005068/206878899-555a0b6e-9c28-4cb2-a6a9-bbf7ebba01ab.png">
<img width="1495" alt="Screenshot 2022-12-11 at 4 31 27 AM" src="https://user-images.githubusercontent.com/84005068/206885293-17cd2702-465a-43a4-aa58-bfa0d2b4d7e4.png">
<img width="1496" alt="Screenshot 2022-12-11 at 12 15 10 AM" src="https://user-images.githubusercontent.com/84005068/206886481-badf098c-599a-4dde-bea7-32d7dc946e13.png">
<img width="1496" alt="Screenshot 2022-12-11 at 12 24 37 AM" src="https://user-images.githubusercontent.com/84005068/206886484-dd832109-5c71-41e8-8d61-b58e7aa9ecde.png">

Reply:
As mentioned, this doesn't need to be signed off as this is the same layout as on the issue form (which has already been agreed)